Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -124,9 +129,21 @@ struct SettingsView: View {
                             } label: {
                                 Text("Delete", comment: "Label to delete a vehicle")
                             }
+                            Button {
+                                isEditingVehicle = true
+                            } label: {
+                                VStack {
+                                    Text("Edit", comment: "Button label to edit this vehicle")
+                                    Image(systemName: SFSymbol.pencil)
+                                }
+                            }
+                        }
+                        .sheet(isPresented: $isEditingVehicle) {
+                                            EditVehicleView(
+                        selectedEvent: $selectedVehicleEvent, viewModel: viewModel)
```

Comment:
Here we'd pass the `vehicleToEdit` because we are using the Vehicle model, not the new one you created